src/auth/digest/Config.cc

```diff
@@ -51,7 +51,6 @@ Helper::ClientPointer digestauthenticators;
 static hash_table *digest_nonce_cache;
```

Comment:
A few lines above, there is an XXX comment quoted below. Please remove or update quoted code because this PR appears to address its "convert to..."  to-do and definitely makes parts of that comment stale:

```C++
/* digest_nonce_h still uses explicit alloc()/freeOne() MemPool calls.
 * XXX: convert to MEMPROXY_CLASS() API
 */
#include "mem/Allocator.h"
#include "mem/Pool.h"
```